torch/library.h

```diff
@@ -611,18 +611,23 @@ class TORCH_API Library final {
   }
 
   /// Declares that for all operators that are subsequently def'ed, their
-  /// abstract impls may be found in the given Python module (pymodule).
-  /// This registers some help text that is used if the abstract impl
+  /// fake impls may be found in the given Python module (pymodule).
+  /// This registers some help text that is used if the fake impl
   /// cannot be found.
   ///
   /// Args:
   /// - pymodule: the python module
   /// - context: We may include this in the error message.
-  Library& impl_abstract_pystub(const char* pymodule, const char* context = "") {
-    impl_abstract_pystub_ = {pymodule, context};
+  Library& has_python_registrations(const char* pymodule, const char* context = "") {
```

Comment:
I would expect that this would return a bool given the function name?

Reply:
TORCH_LIBRARY(lib, m) {
  m.set_python_module("...");
}